ci/install.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Install Miniconda
+if [ $(uname) == Linux ]; then
+    wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+else
+    wget https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -O miniconda.sh
+fi
+bash miniconda.sh -b -p $HOME/miniconda
+source $HOME/miniconda/etc/profile.d/conda.sh
+
+# Make new conda environment with required Python version
+conda create -y -n test python=$PYTHON_VERSION
+conda activate test
+
+# Install dependencies
+conda install -y -q --file requirements/requirements.txt
```

Comment:
Let's change this to

    conda install -c spyder-ide -y -q requirements/conda.txt
    conda install -c spyder-ide -y -q requirements/tests.txt